scripts/Tools/xmlquery

```diff
@@ -127,17 +168,17 @@ def get_value_as_string(case, var, attribute=None, resolved=False, subgroup=None
         value = convert_to_string(value, thistype, var)
     return value
 
-def xmlquery(case, variables, subgroup=None, fileonly=False,
-             resolved=True, raw=False, description=False, group=False,
-             full=False, dtype=False, valid_values=False, xmlfile=None):
+def xmlquery_sub(case, variables, subgroup=None, fileonly=False,
+                 resolved=True, raw=False, description=False, get_group=False,
+                 full=False, dtype=False, valid_values=False, xmlfile=None):
     """
     Return list of attributes and their values, print formatted
 
     """
     results = {}
     comp_classes = case.get_values("COMP_CLASSES")
-    if xmlfile:
-        case.set_file(xmlfile)
+
```

Comment:
I think that this was needed for the --file option.   Please clearly document the case that failed because this was here.   As an example I created a test with the setting of REST_OPTION='nsteps' in env_run.xml and 'none' in env_test.xml.    With this change I cannot view the value in env_test.xml, however I found a bug in master that causes an error with the flag, I'm pushing a change to your branch.